backend/backend/backend-src/routes/events-api.ts

```diff
@@ -2,44 +2,49 @@
  * @file Handlers related to events and balances.
  */
 
-import {Router} from 'express'
-import {handleGetAddressEvents, handleGetAllocations, handleGetBalances, handleGetPackageEvents} from '../lib'
+import { Router } from 'express';
+
+import { handleGetAddressEvents, handleGetAllocations, handleGetBalances, handleGetPackageEvents } from '../lib';
 
 export function createEventsRouter(): Router {
-  const router = Router()
+  const router = Router();
 
   router.get('/balances/:address', async (req, res, next) => {
     try {
-      await handleGetBalances(req, res)
-    } catch (error) {
-      next(error)
+      await handleGetBalances(req, res);
+    }
+    catch (error) {
+      next(error);
     }
-  })
+  });
 
-  router.get('/:address/:ticker', async (req, res, next) => {
+  router.get('/package/:address/:ticker', async (req, res, next) => {
     try {
-      await handleGetPackageEvents(req, res)
-    } catch (error) {
-      next(error)
+      await handleGetPackageEvents(req, res);
+    }
+    catch (error) {
+      next(error);
     }
-  })
+  });
 
-  router.get('/:address', async (req, res, next) => {
+  router.get('/address/:address', async (req, res, next) => {
```

Comment:
@Imamah-Zafar shouldn't this be done in separate PR as it is not related to ESlint